KP add_entry() None password

Greetings,

I'm trying to add an entry with an empty password like that:
```python
entry = kp.add_entry(group, "test", "test", password=None)
```

But I'm getting this error:
```python
.venv/lib/python3.13/site-packages/pykeepass/pykeepass.py:695: in add_entry
    entry = Entry(
.venv/lib/python3.13/site-packages/pykeepass/entry.py:45: in __init__
    E.String(E.Key('Password'), E.Value(password, Protected="True"))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
>   ???
E   TypeError: bad argument type: NoneType(None)
src/lxml/builder.py:219: TypeError
```

The documentation say that the password argument can be `None`:
https://libkeepass.github.io/pykeepass/pykeepass.html#PyKeePass.add_entry

I know that I should use an empty string, but I have to admit that passing `None` would simplify my code a bit.

--- a/pykeepass/entry.py
+++ b/pykeepass/entry.py
@@ -39,10 +39,10 @@ class Entry(BaseElement):
                 expiry_time=expiry_time,
                 icon=icon
             )
-            self._element.append(E.String(E.Key('Title'), E.Value(title)))
-            self._element.append(E.String(E.Key('UserName'), E.Value(username)))
+            self._element.append(E.String(E.Key('Title'), E.Value(title or "")))
+            self._element.append(E.String(E.Key('UserName'), E.Value(username or "")))
             self._element.append(
-                E.String(E.Key('Password'), E.Value(password, Protected="True"))
+                E.String(E.Key('Password'), E.Value(password or "", Protected="True"))
             )
             if url:
                 self._element.append(E.String(E.Key('URL'), E.Value(url)))

--- a/tests/tests.py
+++ b/tests/tests.py
@@ -511,6 +511,19 @@ class EntryTests3(KDBX3Tests):
             self.kp.root_group
         )
 
+    def test_null_values(self):
+        """Set title, username, password to `None` as the docs allow"""
+        entry = Entry(
+            None,
+            None,
+            None,
+            kp=self.kp
+        )
+        self.assertEqual(entry.title, '')
+        self.assertEqual(entry.username, '')
+        self.assertEqual(entry.password, '')
+
+
     def test_references(self):
         original_entry = self.kp.find_entries(title='foobar_entry', first=True)
         original_entry_duplicate = self.kp.find_entries(title='foobar_entry', first=True)

